docs/integrate/lineage/marquez/index.md

```diff
@@ -0,0 +1,258 @@
+(marquez)=
+# Marquez
+
+:::{include} /_include/links.md
+:::
+
+[OpenLineage] is an open source industry standard framework for data lineage. It standardizes the definition of data lineage, the metadata that makes up lineage data, and the approach for collecting lineage data from external systems.
+
+[Marquez] is OpenLineage's lineage repository reference implementation.
+
+Among other tools, OpenLineage integrates with [Airflow] to collect DAG lineage metadata so that inter-DAG dependencies are easily maintained and viewable via a lineage graph, while also keeping a catalog of historical runs of DAGs.
+
+## Data lineage with CrateDB
+
+Today I want to share an example of how we can run Airflow DAGs against a CrateDB database and view lineage data.
+
+### Setup
+
+We will need Docker Compose v2, to install it, make it available to all users in the system, check the installed version, or upgrade from v1, please refer to [the Docker documentation on this topic].
```

Comment:
Just another formatting/layout nit: I don't think we need this nesting on headlines. I think it will be good enough to have `## Setup` and `## Usage` as secondary-level headlines, also to make the new secondary navigation on the right hand side render well.

In this spirit, `## Data lineage with CrateDB` adds a headline level that is effectively unneccessary.